redis_shard.go

```diff
@@ -614,54 +441,46 @@ func getDialOpts(conf RedisShardConfig) []redis.DialOption {
 	if conf.ConnectTimeout != 0 {
 		connectTimeout = conf.ConnectTimeout
 	}
+	opt.ReadTimeout = readTimeout
+	opt.WriteTimeout = writeTimeout
+	opt.DialTimeout = connectTimeout
+	opt.ConnMaxIdleTime = conf.IdleTimeout
```

Comment:
These global timeout options will take effect in every read/write operation.

Reply:
Yep, I already noticed 👍 Think it's enough